### PR TITLE
chore(docs): adding read-only partitions behaviours

### PR DESCRIPTION
**Pull request by amyshwang**
Document https://github.com/questdb/questdb/pull/2710

### CODE REVIEW

File: docs/reference/sql/alter-table-attach-partition.md [hunk 1]
```diff
@@ -28,42 +28,58 @@ This section describes the details of each method.
...
-- _DROP/DETACH PARTITION_: Once the partition folders are unlinked, the symbolic links are removed, but not the content
-  remains. Detaching a partition that was attached via symbolic link does not create a copy `<partition_name>.detached`;
-  in this case, the `DETACH PARTITION` operation is equal to `DROP PARTITION`.
-
-Partitions attached via symbolic links are **read-write** for these operations:
-
-- _INSERT_, _UPDATE_, _ADD/DROP INDEX_, _ADD/DROP/RENAME COLUMN_: Data in the partition folders
-  are modified in place as a result of the operation.
+Partitions attached via the symbolic link approach are **read-only** for the
+following operations: [text](#properties-using-symbolic-links)
```
Comment (marregui) [inline docs/reference/sql/alter-table-attach-partition.md:62]: what is this text anchor for?

**Review by marregui — APPROVED** [review]
LGTM

File: docs/reference/sql/alter-table-attach-partition.md (hunk 1 above)
Comment (amyshwang) [reply docs/reference/sql/alter-table-attach-partition.md]: It's an error - removed...

**Review by marregui — APPROVED** [review]
LGTM

**Pull request merged**